Jenkinsfile_CNP

```diff
@@ -13,29 +13,34 @@ def component = "processor"
 
 def channel = '#bsp-build-notices'
 
-List<LinkedHashMap<String, Object>> apiTestSecrets = [
-  // Base64-encoded PKCS 12 key store, containing client's private key and certificate known to the API
-  secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
-  secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD'),
+def apiTestSecrets = [
+  'bulk-scan-${env}': [
+    // Base64-encoded PKCS 12 key store, containing client's private key and certificate known to the API
+    secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+    secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD'),
 
-  // Base64-encoded PKCS 12 key store containing private key and certificate known to the API, but expired
-  secret('test-expired-key-store', 'TEST_EXPIRED_KEY_STORE'),
-  secret('test-expired-key-store-password', 'TEST_EXPIRED_KEY_STORE_PASSWORD'),
+    // Base64-encoded PKCS 12 key store containing private key and certificate known to the API, but expired
+    secret('test-expired-key-store', 'TEST_EXPIRED_KEY_STORE'),
+    secret('test-expired-key-store-password', 'TEST_EXPIRED_KEY_STORE_PASSWORD'),
 
-  // Base64-encoded PKCS12 key store containing private key and certificate known to the API, but not yet valid
-  secret('test-not-yet-valid-key-store', 'TEST_NOT_YET_VALID_KEY_STORE'),
-  secret('test-not-yet-valid-key-store-password', 'TEST_NOT_YET_VALID_KEY_STORE_PASSWORD'),
+    // Base64-encoded PKCS12 key store containing private key and certificate known to the API, but not yet valid
+    secret('test-not-yet-valid-key-store', 'TEST_NOT_YET_VALID_KEY_STORE'),
+    secret('test-not-yet-valid-key-store-password', 'TEST_NOT_YET_VALID_KEY_STORE_PASSWORD'),
 
-  secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+    secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
 
-  secret('test-storage-account-key', 'TEST_STORAGE_ACCOUNT_KEY')
+    secret('storage-account-primary-key', 'TEST_STORAGE_ACCOUNT_KEY')
+  ]
 ]
 
-
-List<LinkedHashMap<String, Object>> commonSecrets = [
-  secret('test-s2s-secret', 'TEST_S2S_SECRET'),
-  secret('s2s-secret', 'S2S_SECRET'),
```

Comment:
are those gone now?

Reply:
re-read full scope of those secret definitions please :). s2s secrets are retrieved directly from s2s vault and not from masked values inside bulk scan vault